src/core.org

```diff
@@ -618,6 +627,8 @@
         [_ offset] (HSLA. h (m/clamp (+ offset s) 0.0 1.0) l a))
     (adjust-brightness
         [_ offset] (-> _ (as-hsva) (adjust-brightness offset) (as-hsla)))
+    (adjust-luminance
+        [_ offset] (HSLA. h s (m/clamp (+ offset l) 0.0 1.0) a))
```

Comment:
you can use `clamp-normalized` here.

Reply:
Fair play! But it's not used in the rest of the file so wouldn't that just be weird unless the rest of the calls were changed? There's like 50+ instances.

Reply:
heh, that's true. I only left the comment because was spelunking in the code, doing exactly what your PR is about at the same time. 

Maybe @postspectacular wants to chime in - these look like great additions to me!

Reply:
`clamp-normalized` wouldn't be suitable for this here, since that's clipping values to [-1,+1] interval, so current usage is correct...